contract_manager/src/contracts/evm.ts

```diff
@@ -163,9 +268,32 @@ export class EVMContract extends Contract {
     };
   }
 
-  async executeGovernanceInstruction(privateKey: string, vaa: Buffer) {
-    const web3 = new Web3(this.chain.rpcUrl);
-    const { address } = web3.eth.accounts.wallet.add(privateKey);
+  async executeUpdatePriceFeed(senderPrivateKey: string, vaas: Buffer[]) {
+    const web3 = new Web3(this.chain.getRpcUrl());
+    const { address } = web3.eth.accounts.wallet.add(senderPrivateKey);
+    const pythContract = new web3.eth.Contract(EXTENDED_PYTH_ABI, this.address);
+    const priceFeedUpdateData = vaas.map((vaa) => "0x" + vaa.toString("hex"));
+    const updateFee = await pythContract.methods
+      .getUpdateFee(priceFeedUpdateData)
+      .call();
+    const transactionObject =
+      pythContract.methods.updatePriceFeeds(priceFeedUpdateData);
+    const gasEstiamte = await transactionObject.estimateGas({
+      from: address,
+      gas: 100000000,
+      value: updateFee,
+    });
+    return transactionObject.send({
+      from: address,
+      value: updateFee,
+      gas: gasEstiamte * GAS_ESTIMATE_MULTIPLIER,
+      gasPrice: await this.chain.getGasPrice(),
+    });
```

Comment:
if you do this a bunch, consider making a helper function like sendWithGasEstimate